settings.py

```diff
@@ -5,3 +5,5 @@
 # Time (in seconds) to wait retrieve box from storehouse
 BOX_RESTORE_TIMER = 0.1
 CONSISTENCY_INTERVAL = 60
+
+IPV4 = 2048
```

Comment:
1. This is not a setting, this is a constant.
1. It's not used anywhere else, so it can be kept inside the original file (match.py).
1. The name `IPV4` doesn't say what it is, please try to be more descriptive (e.g.: IPV4_ETH_TYPE)

Reply:
Changed at commit 47c31e8